src/@chakra-ui/gatsby-plugin/components/Table.ts

```diff
@@ -0,0 +1,53 @@
+import { ComponentStyleConfig } from "@chakra-ui/react"
+
```

Comment:
Table line height is too small here. 

Before:

<img width="972" alt="Screenshot 2022-10-03 at 11 00 28" src="https://user-images.githubusercontent.com/62268199/193551317-ea2b2f5a-4b1e-457a-8482-502e36169e9e.png">

This PR: 

<img width="972" alt="Screenshot 2022-10-03 at 11 00 25" src="https://user-images.githubusercontent.com/62268199/193551331-0473e192-23ad-4884-b1da-397cf381e343.png">